src/IndentRainbow.Extension/Options/DefaultRainbowIndentOptions.cs

```diff
@@ -11,13 +11,22 @@ public enum HighlightingMode
         Monocolor = 1
     }
 
+    public enum ColorMode
+    {
+        [Description("Hard")]
+        Hard = 0,
```

Comment:
I think this should be named `Solid` instead of `Hard` since we either render a SolidColorbrush or a LinearGradientBrush. The word hard might not be something every user knows or is familiar with.

Reply:
Totally agree! `Solid` is the better option here.

Reply:
You will also need to rename the property, for some reason the Visual Studio settings page does not care about the description property but rather the actual enum value.

Reply:
Agree! Even it would be working like this, this should be changed for consistency.